src/request/create/index.html

```diff
@@ -80,9 +80,6 @@
         <div id="choose-identicon" class="page nq-card">
             <div class="page-header nq-card-header">
                 <div class="progress-indicator"></div>
-                <a tabindex="0" class="page-header-back-button">
-                    <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-arrow-left"/></svg>
-                </a>
                 <h1 data-i18n="create-heading-choose-identicon" class="nq-h1">Choose an Avatar</h1>
```

Comment:
But what if the request comes from the onboarding menu? Then a back button would be valid.

Reply:
That is true. We do not have a back arrow for login requests, too.